open-in-vscode/src/open-in-vscode.c

```diff
@@ -111,36 +157,18 @@ GList *menuProviderGetSelectedItems(NautilusMenuProvider *provider, GtkWidget *w
   }
 
   char buttonLabel[32];
+  size_t nrOfDirs = 0ul;
   size_t nrOfFiles = 0ul;
 
-  for (GList *element = files; element != NULL; element = element->next) {
-    NautilusFileInfo *file = element->data;
-
-    if (nautilus_file_info_is_mime_type(file, "text/*"))
-    {
-      char *absoluteFilePath = g_file_get_path(nautilus_file_info_get_location(file));
-
-      command = addToCommand(command, absoluteFilePath);
-      assert(command);
-      ++nrOfFiles;
-
-      g_free(absoluteFilePath);
-    }
-  }
+  handleMimeType("inode/*", &command, files, &nrOfDirs);
```

Comment:
Links to files are `inode/symlink` and thus would be falsely interpreted as directories

Reply:
symlinks are a general edge case right here